tests/bsim/bluetooth/host/adv/extended/testcase.yaml

```diff
@@ -0,0 +1,20 @@
+common:
+  build_only: true
+  tags:
+    - bluetooth
+  platform_allow:
+    - nrf52_bsim
+    - nrf5340bsim/nrf5340/cpunet
+  harness: bsim
```

Comment:
It would have been nice to see at least 1 test for the nRF5340 cpuapp as well, to act as a reference for how to use this with sysbuild/dual CPU